loadgen/package.json

```diff
@@ -30,10 +30,12 @@
     "@agoric/install-ses": "*",
     "@agoric/marshal": "*",
     "@agoric/notifier": "*",
+    "@agoric/promise-kit": "*",
     "@agoric/store": "*",
     "@agoric/ui-components": "*",
     "@agoric/zoe": "*",
-    "esm": "^3.2.5"
+    "esm": "^3.2.5",
+    "firebase": "^9.0.0-beta.8"
```

Comment:
we need an unreleased version of firebase? that's.. exciting

Reply:
Yeah, v9 is modular and avoid loading everything, which I figured would be less problematic in a SES environment.